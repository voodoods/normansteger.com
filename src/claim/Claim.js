import React, { Component } from 'react'
import StyledClaim from './Claim.styled'
import Button from '../button/Button'

class Home extends Component {
  render() {
    return (
      <StyledClaim className="Claim">
        <div className="pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1">
          <h3>I create Web Applications</h3>
          <span>User Friendly. Performant. Responsive. Test Driven.</span>

          <div className="seperator">
            <Button text="Find out more" route="/skills" primary={true} />
          </div>
        </div>
      </StyledClaim>
    )
  }
}

export default Home
