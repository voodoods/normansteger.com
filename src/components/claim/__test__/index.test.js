import React from 'react'
import { shallow } from 'enzyme'
import Claim from '../'

describe('<Claim /> component index', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<Claim />).dive()
    expect(wrapper).toMatchSnapshot()
  })
})
