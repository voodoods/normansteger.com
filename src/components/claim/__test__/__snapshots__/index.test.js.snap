// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Claim /> component index expects component to match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ForwardRef
    className="Claim"
  >
    <div
      className="pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1"
    >
      <h3>
        I create Web Applications
      </h3>
      <span>
        User Friendly. Performant. Responsive. Test Driven.
      </span>
      <div
        className="seperator"
      >
        <withProps(Button)
          primary={true}
          route="/skills"
          text="Find out more"
        />
      </div>
    </div>
  </ForwardRef>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1"
      >
        <h3>
          I create Web Applications
        </h3>
        <span>
          User Friendly. Performant. Responsive. Test Driven.
        </span>
        <div
          className="seperator"
        >
          <withProps(Button)
            primary={true}
            route="/skills"
            text="Find out more"
          />
        </div>
      </div>,
      "className": "Claim",
      "forwardedComponent": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  .pure-u-lg-3-5 {
    height: 90%;
    padding: 2em 0;
  }

  h3 {
    font-size: 3em;
    margin-bottom: 0.5em;
  }

  span {
    font-weight: 300;
    font-size: 1.6em;
    color: #aaa;
  }

  .seperator {
    margin-top: 3em;
  }

  @media (max-width: 768px) {
    .pure-u-md-1,
    .pure-u-sm-1 {
      padding: 1em;
      text-align: center;
    }
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
      "forwardedRef": null,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            I create Web Applications
          </h3>,
          <span>
            User Friendly. Performant. Responsive. Test Driven.
          </span>,
          <div
            className="seperator"
          >
            <withProps(Button)
              primary={true}
              route="/skills"
              text="Find out more"
            />
          </div>,
        ],
        "className": "pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "I create Web Applications",
          },
          "ref": null,
          "rendered": "I create Web Applications",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "User Friendly. Performant. Responsive. Test Driven.",
          },
          "ref": null,
          "rendered": "User Friendly. Performant. Responsive. Test Driven.",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <withProps(Button)
              primary={true}
              route="/skills"
              text="Find out more"
            />,
            "className": "seperator",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "primary": true,
              "route": "/skills",
              "text": "Find out more",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1"
        >
          <h3>
            I create Web Applications
          </h3>
          <span>
            User Friendly. Performant. Responsive. Test Driven.
          </span>
          <div
            className="seperator"
          >
            <withProps(Button)
              primary={true}
              route="/skills"
              text="Find out more"
            />
          </div>
        </div>,
        "className": "Claim",
        "forwardedComponent": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "rules": Array [
              "
  .pure-u-lg-3-5 {
    height: 90%;
    padding: 2em 0;
  }

  h3 {
    font-size: 3em;
    margin-bottom: 0.5em;
  }

  span {
    font-weight: 300;
    font-size: 1.6em;
    color: #aaa;
  }

  .seperator {
    margin-top: 3em;
  }

  @media (max-width: 768px) {
    .pure-u-md-1,
    .pure-u-sm-1 {
      padding: 1em;
      text-align: center;
    }
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
        "forwardedRef": null,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              I create Web Applications
            </h3>,
            <span>
              User Friendly. Performant. Responsive. Test Driven.
            </span>,
            <div
              className="seperator"
            >
              <withProps(Button)
                primary={true}
                route="/skills"
                text="Find out more"
              />
            </div>,
          ],
          "className": "pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "I create Web Applications",
            },
            "ref": null,
            "rendered": "I create Web Applications",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "User Friendly. Performant. Responsive. Test Driven.",
            },
            "ref": null,
            "rendered": "User Friendly. Performant. Responsive. Test Driven.",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <withProps(Button)
                primary={true}
                route="/skills"
                text="Find out more"
              />,
              "className": "seperator",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "primary": true,
                "route": "/skills",
                "text": "Find out more",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
